### PR TITLE
Ddcnl 1433 Play Upgrade

### PR DESCRIPTION
**Pull request by AndrewArkless**

### CODE REVIEW

File: app/controllers/controllershelpers/ControllerLikeHelpers.scala [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package controllers.helpers
+package controllers.controllershelpers
```
Comment (oc-ste) [inline app/controllers/controllershelpers/ControllerLikeHelpers.scala]: If you delete:

`implicit def hc(implicit request: Request[_]): HeaderCarrier =
    HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))	    HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))`

You can remove the lines:
`implicit val hc = HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))`

The reason for this is in the `FrontendController` it does this implicit conversion already, which leads to an ambiguous implicit conversion for HeaderCarriers.

File: app/controllers/PertaxBaseController.scala [hunk 2]
```diff
@@ -16,15 +16,16 @@
...
 import play.api.mvc._
-import uk.gov.hmrc.play.bootstrap.controller.Utf8MimeTypes
+import uk.gov.hmrc.play.bootstrap.controller.{FrontendController, Utf8MimeTypes}
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 
-abstract class PertaxBaseController extends Controller with Utf8MimeTypes with I18nSupport with ControllerLikeHelpers {
+abstract class PertaxBaseController(cc: MessagesControllerComponents)(implicit ec: ExecutionContext)
+    extends FrontendController(cc) with Utf8MimeTypes with I18nSupport with ControllerLikeHelpers {
```
Comment (oc-ste) [inline app/controllers/PertaxBaseController.scala]: Can delete `Utf8MimeTypes`

**Comment by oc-ste** [comment]
Good work @AndrewArkless and @brendanstraughan

**Comment by oc-ste** [comment]
Reopened as branch ddcnl-1433-a

**Comment by jpatton638** [comment]
@oc-ste is this meant to be closed?

**Comment by oc-ste** [comment]
Yeah check out https://github.com/hmrc/pertax-frontend/pull/405 for the new PR, it was easier to create a new branch than delete off some changes

**Pull request closed without merging**